sklearnex/tests/_utils_spmd.py

```diff
@@ -179,7 +182,11 @@ def _assert_kmeans_labels_allclose(
         AssertionError: If clusters are not correctly assigned.
     """
 
+    spmd_labels_as_numpy = _as_numpy(spmd_labels)
+    spmd_centers_as_numpy = _as_numpy(spmd_centers)
```

Comment:
```suggestion
    np_spmd_labels = _as_numpy(spmd_labels)
    np_spmd_centers = _as_numpy(spmd_centers)
```
Shortening without lost of information